implementations/rust/ockam/ockam_command/tests/bats/load/orchestrator.bash

```diff
@@ -21,16 +21,9 @@ function skip_if_long_tests_not_enabled() {
 }
 
 function load_orchestrator_data() {
-  if [ ! -z "${ORCHESTRATOR_TESTS}" ]; then
-    if [ ! -f "$OCKAM_HOME_BASE/project.json" ]; then
-      OCKAM_HOME=$OCKAM_HOME_BASE $OCKAM project information --output json >"$OCKAM_HOME_BASE/project.json"
-    fi
-    export PROJECT_JSON_PATH="$OCKAM_HOME_BASE/project.json"
-  fi
-}
-
-function copy_orchestrator_data() {
   if [ ! -z "${ORCHESTRATOR_TESTS}" ]; then
     cp -a $OCKAM_HOME_BASE $OCKAM_HOME
+    $OCKAM project information --output json >"$PROJECT_JSON_PATH"
```

Comment:
Shouldn't those 2 statements be switched?

Reply:
Yes I was talking about L.26 and L27, I should have been more precise. Maybe now I would write this instead to avoid the repetition:
```sh
export PROJECT_JSON_PATH="$OCKAM_HOME/project.json"
$OCKAM project information --output json >"$PROJECT_JSON_PATH"
```

Reply:
🤦 of course, fixed!